### ISSUE TITLE
Links enclosed in < > not linked or rendered correctly

### ISSUE THREAD

**Issue by nigelsim** [microsoft/vscode-markdown-tm-grammar#66]
The following Markdown should produce a link that automatically escapes the space in the link. 

```
[link text](</file im linking to.md>)
```

However, it is not possible to follow this link in the editor, and the colourising is incorrect.

![Screenshot from 2019-12-22 15-43-28](https://user-images.githubusercontent.com/2869/71317770-40f8ca00-24d2-11ea-8b76-d2da05da717c.png)

This does however link correctly in the VS Code Markdown Preview.

![preview-link](https://user-images.githubusercontent.com/2869/71317784-638ae300-24d2-11ea-8a53-3a5d2aa9dfc5.png)

This definition has changed in [0.29](https://spec.commonmark.org/0.29/#link-destination). The relevant [Common Mark testcase](https://spec.commonmark.org/0.29/spec.json) is as follows:
```
  {
    "markdown": "[link](</my uri>)\n",
    "html": "<p><a href=\"/my%20uri\">link</a></p>\n",
    "example": 486,
    "start_line": 7543,
    "end_line": 7547,
    "section": "Links"
  },
  ```

**Comment by mjbvz** [comment]
This extension only handles the grammar highlighting, not the link providing logic (but as you note the highlighting is also incorrect)

Please file the link detection issue against VS Code itself

**Comment by rahul0705** [comment]
This looks like a duplicate of #80 which provides additional detail?

**Comment by mjbvz** [comment]
Fixed by 2f9bda284d391f97982f7932ba49eb89b2ac83b1
